flake.nix

```diff
@@ -67,9 +67,7 @@
       legacyPackages = forAllSystems (
         { pkgs, ... }:
         {
-          nixosTests = pkgs.callPackage ./nixos/tests/all-tests.nix {
-            nixos-lib = import (nixpkgs + "/nixos/lib") { };
-          };
+          nixosTests = import ./nixos/tests/all-tests.nix pkgs;
```

Comment:
No keep it `callPackage`, and, since we no longer have to fill `hostPkgs`, replace the formal arguments with `{ testers }:`